main.go

```diff
@@ -388,6 +404,9 @@ func loadConfig(filename string, queryUserData bool) (*Config, error) {
 		TasksDir:                       defaultTasksDir(),
 		WorkerGroup:                    "test-worker-group",
 		WorkerTypeMetadata:             map[string]interface{}{},
+		Project:                        "",
+		LogLevel:                       "debug",
+		Tags:                           "",
```

Comment:
Should tags be a string array, rather than a string?

Also, all these config variables need to be defined in the help text at the top of `main.go`. Note config is defined in two sections - mandatory and optional parameters - make sure to add to the correct section.